### ISSUE TITLE
How I compile this project in Xcode

### ISSUE THREAD

**Issue by nassifbasheer** [elucideye/drishti#756]
any help please

**Comment by ruslo** [comment]
Can you please explain what you have tried and what error you've got?

See also:
- https://github.com/elucideye/drishti#get-latest-sources
- https://github.com/elucideye/drishti#generate-and-build

**Comment by nassifbasheer** [comment]
I cloned the repo, moved to drishti location and followed the next two steps
cmake -H. -B_builds -GXcode -DHUNTER_STATUS_DEBUG=ON -DDRISHTI_BUILD_EXAMPLES=ON
cmake --build _builds --config Release

Build was successful , but no xcodeproject was build. I am missing something

**Comment by ruslo** [comment]
> cmake --build _builds --config Release

If you're able to run this command successfully, then Xcode project is generated. Check for the `_builds/drishtisdk.xcodeproj`.

**Comment by nassifbasheer** [comment]
My mistake , I was checking in src/examples.  Can we run this xcodeproject on an iOS device

**Comment by ruslo** [comment]
> Can we run this xcodeproject on an iOS device

No, default Xcode project is targeting macOS. If you want to build for iOS, you should use toolchain. Start from here:
- https://github.com/elucideye/drishti#ios

**Comment by nassifbasheer** [comment]
Thanks, so I guess , 
I need to do the following steps

cd  Users/nassif/Desktop/RootFolder/drishti/src/lib/drishti
mkdir build
cd build
cmake .. -G Xcode -DCMAKE_TOOLCHAIN_FILE=../../ios.toolchain.cmake -DPLATFORM=OS64COMBINED
cmake --build . --config Release --target install

**Comment by ruslo** [comment]
> so I guess, I need to do the following steps

No, there are no such instructions in README.

Please check:
- https://github.com/elucideye/drishti#polly-based-build
- https://github.com/elucideye/drishti#ios-build
- https://github.com/elucideye/drishti#choosing-simplest-toolchain
